ardublockly/index.html

```diff
@@ -13,11 +13,11 @@
   <link rel="stylesheet" href="ardublockly.css" media="screen,projection">
 
   <!-- Ardublockly - These three files contain the compress version -->
-  <script src="../blockly/blockly_compressed.js"></script>
-  <script src="../blockly/blocks_compressed.js"></script>
-  <script src="../blockly/arduino_compressed.js"></script>
+  <!--<script src="../blockly/blockly_compressed.js"></script>-->
+  <!--<script src="../blockly/blocks_compressed.js"></script>-->
+  <!--<script src="../blockly/arduino_compressed.js"></script>-->
```

Comment:
While this is necessary for testing I would prefer to maintain the compressed versions in the codebase (on commit the CI servers will automatically build blockly including all the changes).